addons/website_sale/models/product_product.py

```diff
@@ -46,7 +46,10 @@ def _check_base_unit_count(self):
     def _compute_product_website_url(self):
         for product in self:
             attributes = ','.join(str(x) for x in product.product_template_attribute_value_ids.ids)
-            product.website_url = "%s#attr=%s" % (product.product_tmpl_id.website_url, attributes)
+            url = product.product_tmpl_id.website_url
+            if attributes:
+                url += "#attr=%s" % attributes
```

Comment:
Why not using `url_join` ?
```suggestion
                url += url_join(URL, "#attr=%s" % attributes, allow_fragments=True)
```

Reply:
not sure it is more readable... and don't add plus-value from what I know in this case, since we know the right part, and the left part is not a user input... 


`+= -> =`=> since we provide already the first part
`no allow_fragment` => the default value is True (verbose is not better than implicit)

but ok...